apps/session-manager/src/controllers/pagopa.ts

```diff
@@ -0,0 +1,127 @@
+import {
+  IResponseErrorNotFound,
+  IResponseErrorTooManyRequests,
+  IResponseErrorValidation,
+  IResponseSuccessJson,
+  ResponseErrorValidation,
+  ResponseSuccessJson,
+} from "@pagopa/ts-commons/lib/responses";
+import * as TE from "fp-ts/TaskEither";
+import * as RTE from "fp-ts/ReaderTaskEither";
+import { pipe } from "fp-ts/function";
+import * as O from "fp-ts/Option";
+import { EmailString } from "@pagopa/ts-commons/lib/strings";
+import { FnAppAPIRepositoryDeps } from "../repositories/fn-app-api";
+import { WithExpressRequest } from "../utils/express";
+import { WithUser } from "../utils/user";
+import { ProfileService, RedisSessionStorageService } from "../services";
+import { EmailAddress } from "../generated/backend/EmailAddress";
+import { InitializedProfile } from "../generated/backend/InitializedProfile";
+import { RedisRepositoryDeps } from "../repositories/redis";
+import { PagoPAUser } from "../generated/pagopa/PagoPAUser";
+
+const VALIDATION_ERROR_TITLE = "Validation Error";
+type PagoPAGetUserHandler = RTE.ReaderTaskEither<
+  { enableNoticeEmailCache: boolean } & FnAppAPIRepositoryDeps &
+    RedisRepositoryDeps &
+    WithUser &
+    WithExpressRequest,
+  Error,
+  | IResponseErrorValidation
+  | IResponseErrorNotFound
+  | IResponseErrorTooManyRequests
+  | IResponseSuccessJson<PagoPAUser>
+>;
+
+export const getUser: PagoPAGetUserHandler = (deps) => {
+  const getProfileAndSaveNoticeEmailCache = pipe(
+    ProfileService.getProfile(deps),
+    // TODO: remove this error remap? it was found in io-backend
+    TE.mapLeft((_) => Error("Internal server error")),
```

Comment:
In the backend the `getProfile` returns a `Promise` and the error maps the throw scenario. In here we already have an error on the left path. Maybe do we want to change the error message?

Reply:
I left the same error message with a comment in https://github.com/pagopa/io-auth-n-identity-domain/pull/55/commits/b46301c88ae27cb4314acaf32e81c678ce5bd444